CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.10.0 (Released 2025-03-04)
```

Comment:
looks like there's a few more updates we should include here -- e.g. https://github.com/moovfinancial/moov-go/commit/88b7916e1949449cc4aa12e21b7069f0f646b935

Reply:
done